website/source/docs/upgrade/index.html.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: "docs"
+page_title: "Upgrade Nomad"
+sidebar_current: "docs-upgrade-upgrading"
+description: |-
+  Learn how to upgrade Nomad.
+---
+
+# Upgrading Nomad
+
+Both Nomad Clients and Servers are meant to be long-running processes that
+maintain communication with each other. Nomad Servers maintain quorum with other
+Servers and Clients are in constant communication with Servers. As such, care
+should be taken to properly upgrade Nomad to ensure minimal service disruption.
+
+This page documents how to upgrade Nomad when a new version is released.
+
+## Standard Upgrades
+
+For upgrades we strive to ensure backwards compatibility. For most upgrades, the
+process is simple. Assuming the current version of Nomad is A, and version B is
+released.
+
+1. On each server, install version B of Nomad.
```

Comment:
I think we shouldn't say "On each server" since deployments can be different on cloud environments than private data centers.

Consider saying, "Bring up the new Nomad Server cluster" or "Bring up the cluster with version B of Nomad Server".